aspnet/migration/rc1-to-rc2.rst

```diff
@@ -76,15 +78,27 @@ IHostingEnvironment changes
 
 All environment variables are now prefixed with the ``ASPNETCORE_`` prefix.
 
-======================  ===========================
-Old prefix              New prefix
-======================  ===========================
-ASPNET\_WEBROOT         ASPNETCORE\_WEBROOT
-ASPNET\_SERVER          ASPNETCORE\_SERVER
-ASPNET\_APP             ASPNETCORE\_APPLICATIONNAME
-ASPNET\_ENVIRONMENT     ASPNETCORE\_ENVIRONMENT
-ASPNET\_DETAILEDERRORS  ASPNETCORE\_DETAILEDERRORS
-======================  ===========================
+===================================================  ===================================================  
+RC1                                                  RC2                           
+===================================================  ===================================================
+ASPNET_APP, ASPNET_APPLICATIONNAME, Hosting:App      ASPNETCORE_APP, ASPNETCORE_APPLICATIONNAME
+ASPNET_STARTUPASSEMBLY                               ASPNETCORE_STARTUPASSEMBLY
+ASPNET_DETAILEDERRORS, Hosting:DetailedErrors        ASPNETCORE_DETAILEDERRORS
+ASPNET_ENVIRONMENT, ASPNET_ENV, Hosting:Environment  ASPNETCORE_ENVIRONMENT, ASPNETCORE_ENV
```

Comment:
We do not read `ASPNETCORE_ENV` in RC2. However we still support `Hosting:Environment` and `ASPNET_ENV`. Note that these are marked as obsolete and the user will see the message `The environment variable 'Hosting:Environment' is obsolete and has been replaced with 'ASPNETCORE_ENVIRONMENT'` if they use it.